src/devices/thermostat.cpp

```diff
@@ -843,6 +843,8 @@ void Thermostat::process_RC35wwTimer(std::shared_ptr<const Telegram> telegram) {
     if (telegram->type_id != 0x38) {
         return;
     }
+
+    // TODO: should this be >= 87 to avoid out of bounds?
```

Comment:
No, this is the right check, we make sure that all offsets 87-92 are in the telegram. (e.g. read 10 38 90 gives offsets 90-117, so 87, 88, 89 are missing, o processing)

Reply:
yes sorry, not sure what I was thinking